File: web/src/app/shared/customModels/schedule-interview-model.ts
import { ScheduleUserForCandidateModel } from '../../webapi/models/schedule-user-for-candidate-model';
import { UserViewModel } from '../../webapi/models/user-view-model';

export class ScheduleInterviewModel {
    finishedEventOrder?: number;
    nextApprovalEvent?: number;
    scheduleUserForCandidateModelList?: ScheduleUserForCandidateModel[];
    scheduledDate?: Date;
    users?: UserViewModel[];
    interviewScheduledTime?: Date[];
}

File: api/RS.API/RS.Model/Candidate/scheduleInterviewModel.cs
using RS.ViewModel.Approval;
using RS.ViewModel.User;
using System;
using System.Collections.Generic;
using System.Text;

namespace RS.ViewModel.Candidate
{
    public class ScheduleInterviewModel
    {
        public int FinishedEventOrder { get; set; }

        public int NextApprovalEvent { get; set; }

        public List<ScheduleUserForCandidateModel> ScheduleUserForCandidateModelList { get; set; }

        public DateTime ScheduledDate { get; set; }

        public List<UserViewModel> Users { get; set; }

        public List<DateTime> InterviewScheduledTime { get; set; }

    }
}
